lib/manifest.js

```diff
@@ -290,6 +290,18 @@ internals.manifest = [
         list: true,
         useFilename: internals.passthruOn('name')
     },
+    {
+        place: 'events',
+        method: 'event',
+        signature: ['events'],
+        list: true
+    },
+    {
+        place: 'event-listeners',
+        method: 'on',
+        signature: ['criteria', 'listener'],
+        list: true
+    },
```

Comment:
If we want to get fancy, both of these can `useFilename` to hint at the event name being referenced.